frontend/src/styles/global.scss

```diff
@@ -673,6 +673,13 @@ body {
                 }
             }
         }
+
+        h1,
+        h2,
+        h3,
+        h4 {
+            font-family: var(--font-title);
+        }
```

Comment:
I had to switch to this (vs `body .posthog-3000 &`) because some sidebar stuff (for now) is dependent on `body h1` and doing our normal nested `.posthog-3000 &` inside of another rule` didn't apply to them because `.posthog-3000 body h1` doesn't work.

Lots of easy cleanup once we sunset Fisher-Price!